### PR TITLE
Documentation for data parallel DeepXDE

### PR DESCRIPTION
**Pull request by pescap**

### CODE REVIEW

File: docs/user/parallel.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+Parallel training
```
Comment (lululxvi) [inline docs/user/parallel.rst:1]: Parallel Training

File: docs/user/parallel.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+=================
+
+Introduction
+------------
+
+DeepXDE utilizes data-parallel acceleration through the `Horovod <https://horovod.readthedocs.io/>`_ training framework. To compensate for the memory shortcomings of GPUs, batch size can be increased while employing data-parallel acceleration.
+
+For :math:`\textrm{size}` GPUs and :math:`\textrm{rank}=1, \cdots, \mathrm{size}`, the data-parallel approach is as follows:
+- We define a dataset :math:`\mathcal{T}_\textrm{rank}` for each rank;
```
Comment (lululxvi) [inline docs/user/parallel.rst:10]: Wrong format. Please see the generated doc.

**Comment by pescap** [comment]
I cannot find the updated ReadTheDocs posterior to these new commits...

Locally, `make html` behaved perfect on my computer, but the RDD run was not ok..

**Comment by lululxvi** [comment]
Here, https://deepxde--1351.org.readthedocs.build/en/1351/

**Pull request merged**